app/celery/broadcast_message_tasks.py

```diff
@@ -111,6 +111,24 @@ def send_broadcast_event(broadcast_event_id):
         return
 
     broadcast_event = dao_get_broadcast_event_by_id(broadcast_event_id)
+
+    broadcast_message = broadcast_event.broadcast_message
+    # raise a P1 to alert team that broadcast is going out.
+    message = '\n'.join([
+        'Broadcast Sent',
+        '',
+        f'https://www.notifications.service.gov.uk/services/{broadcast_message.service_id}/current-alerts/{broadcast_message.id}',  # noqa
+        '',
+        f'This broacast has been sent on channel {broadcast_message.service.broadcast_channel}.',
+        f'This broadcast is targeted at areas {broadcast_message.areas.get("areas")}.',
```

Comment:
Can we link to the runbook google doc here which includes instructions for what to do if an alert goes out accidentally.

Probably worth saying in the zendesk ticket, if the alert going out is unexpected then please follow runbook for accidental alert or similar...

Also given this alert could go off and need a response out of hours, we probably need to think about bringing Unboxed up to speed in the next week or two